### PR TITLE
added initial bootup sequence diagram for project manager overview. g…

### PR DESCRIPTION
**Pull request by tkothadev**
## Change summary

This is an initial commit of a larger initiative for the Project Manager EngineDev documentation. The first step is to provide a high level overview graph of how the Project Manager works.

To that end, this change introduces a boot up sequence that outlines the key steps required for the Project Manager to begin running. This graph also shows the core classes used for managing screens and transitions.

To aid in codebase comprehension, relevant github links are embedded in each node on the graph, pointing to the relevant code segment.

Changes can be found at `docs/engine-dev/tools/project-manager/` on the test server.

### Submission Checklist:

* [x] **Descriptive active voice** - Do descriptive sentences have a clear *subject* and *action verb*?
* [x] **Answer the question at hand** - Does the documentation answer a *what*, *why*, *how*, or *where* type of question?
* [x] **Consistency** - Does the content consistently follow the [Style Guide](https://o3de.org/docs/contributing/to-docs/style-guide/quick-reference)?
* [x] **Help the user** - Does the documentation show the user something *meaningful*?

### CODE REVIEW

**Comment by tkothadev** [comment]
The resulting graph should look like the following:
Screen Shot 2023-01-27 at 9 52 24 AM

**Review by micronAMZN — APPROVED** [review]

**Comment by AMZN-alexpete** [comment]
It might be good to also add a README.md in the o3de repo code/Tools/ProjectManager folder that points to this documentation existing.

**Review by cgalvan — APPROVED** [review]

**Review by AMZN-alexpete — APPROVED** [review]

**Comment by tkothadev** [comment]
@AMZN-alexpete good callout with the README. Do we want to wait till the docs are of sufficient quality and released in main branch? Right now engine dev docs are locked away in development branch, so it may not be as helpful atm.

**Comment by AMZN-alexpete** [comment]
> @AMZN-alexpete good callout with the README. Do we want to wait till the docs are of sufficient quality and released in main branch? Right now engine dev docs are locked away in development branch, so it may not be as helpful atm.

Yes, waiting till these docs are a sufficient quality is good idea but don't wait till they're in the main branch - users can select the "developer" version of the docs to see it 
image

**Comment by tkothadev** [comment]
oh nice, I didn't know development docs were already available

File: content/docs/engine-dev/tools/project-manager/_index.md [hunk 1]
```diff
@@ -1,13 +1,160 @@
...
 
 {{< note >}}
 This information is for developers of the **Project Manager** tool. If you're a user working with Project Manager, please refer to the [Project Manager User Guide](/docs/user-guide/project-config/project-manager).
 {{< /note >}}
 
-{{< todo >}}
-This is a stub article intended to serve as an example of guide structure, and is not necessarily content that will be made available in the near future.
-{{< /todo >}}
+
+```mermaid
```
Comment (willihay) [inline content/docs/engine-dev/tools/project-manager/_index.md:12]: Make a note of the links in the graph nodes to increase awareness of their availability.
```suggestion
{{< tip >}}
Click on a node to open the relevant source code for that node.
{{< /tip >}}

```mermaid
```

**Review by willihay — CHANGES_REQUESTED** [review]

**Pull request merged**